packages/ui-kit/src/components/SimpleFilter/SimpleFilter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { SyntheticEvent, useRef } from 'react'
+import { useTopValues } from '../../hooks'
+import { FilterInput, FilterOperator, getTimeZone } from '../../helpers'
+
+import { Autocomplete } from '../Autocomplete'
+import { AutocompleteOption } from '../Autocomplete/Autocomplete.types'
+import { useFilter } from '../FilterProvider/useFilter'
+
+import { SimpleFilterProps } from './SimpleFilter.types'
+import { withContainer } from '../withContainer'
+import { ErrorFallback } from '../ErrorFallback'
+import { Loader } from '../Loader'
+import componentStyles from './SimpleFilter.module.scss'
+
+const SimpleFilterComponent = ({
+  autocompleteProps,
+  columnName: columnNameProp = '',
+  query,
+  errorFallbackProps,
+  error,
+  loading,
+  loaderProps,
+  options = []
+}: SimpleFilterProps) => {
+  const id = useRef(Symbol()).current
```

Comment:
This is probably the simplest way to create an unique identifier for the filter list, as long as we don't use the filters as part of the UI everything should be fine but lmk if you would like to use a different approach.